commonui/src/main/java/com/hyperwallet/android/ui/common/util/ErrorTypes.java

```diff
@@ -0,0 +1,66 @@
+package com.hyperwallet.android.ui.common.util;
+
+import static com.hyperwallet.android.ExceptionMapper.EC_AUTHENTICATION_TOKEN_PROVIDER_EXCEPTION;
+import static com.hyperwallet.android.ExceptionMapper.EC_IO_EXCEPTION;
+import static com.hyperwallet.android.ExceptionMapper.EC_JSON_EXCEPTION;
+import static com.hyperwallet.android.ExceptionMapper.EC_JSON_PARSE_EXCEPTION;
+import static com.hyperwallet.android.ExceptionMapper.EC_UNEXPECTED_EXCEPTION;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class ErrorTypes {
+    // Error Tags
+    public static final String ERROR_CODE = "error_code";
+    public static final String ERROR_TYPE = "error_type";
+    public static final String ERROR_MESSAGE = "error_message";
+    public static final String ERROR_FIELD_NAME = "erfd";
+    public static final String ERROR_DESCRIPTION = "error_description";
```

Comment:
we don't need this error constants from line 14~18 please remove this and use Insight constants instead